config/sanctum_orm.php

```diff
@@ -22,17 +22,4 @@
         ],
```

Comment:
The folder name changed from `Config` to `config`, all the other folder names are starting a capital letter